crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -470,22 +469,6 @@ impl LedgerDB {
         Ok(())
     }
 
-    /// Used by the sequencer to record that it has committed to soft confirmations on a given L2 height
-    #[instrument(level = "trace", skip(self), err, ret)]
```

Comment:
we can also remove the table and everything related to it

and rename `LastSequencerCommitmentSentL2`to just `LastSequencerCommitmentSent`

Reply:
Done.